### PR TITLE
Bison Coin

### PR DESCRIPTION
**Pull request by BisoncoinADA**
$Bison Coin is a decentralized meme coin on cardano network. Its a High potential, Formally verified and Peer reviewed meme coin for Cardano degans. Intended to build casino game. We are indeed Inspired by master Charles love of Bisons.

From a land far, far away… Beyond the realms of cardano Forest there is rumoured Big bison herd grazing in memeverse.

1000 trillion Bisons grazing on memes looking for Cardano Degans to Farm em.

You can adopt em through Ada, Earn em through steak pools, bet on em on bison Fury.

When you eat em you will be high as F because they are fed by formally verified pure weed!!

We ensure Meme maximalism, will provide 100% Fair opportunity for All Ranchers . Bisons believe that "Meme + utility = mooon" 

Hello fellow Cardano Ranchers welcome to bison farm

### CODE REVIEW

File: tokens/BisonCoin [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "policyId": "1774343241680e4daef7cbfe3536fc857ce23fb66cd0b66320b2e3dd",
+    "name": "BISON COIN",
```
Comment (MuesliSwapTeam) [inline tokens/BisonCoin:3]:
```suggestion
    "name": "BISON", 
```

This should be the asset name. If I see correctly, the asset name for your token is just "BISON"

**Comment by MuesliSwapTeam** [comment]
Thank you for the submission! Please resolve the open issues and then we can go ahead and list it :)

**Comment by BisoncoinADA** [comment]
OK

**Comment by BisoncoinADA** [comment]
Submitted again

**Pull request closed without merging**